src/search/landmarks/landmark_heuristic.h

```diff
@@ -0,0 +1,64 @@
+#ifndef LANDMARKS_LANDMARK_HEURISTIC_H
+#define LANDMARKS_LANDMARK_HEURISTIC_H
+
+# include "../heuristic.h"
+
+class BitsetView;
+
+namespace successor_generator {
+class SuccessorGenerator;
+}
+
+namespace landmarks {
+class LandmarkFactory;
```

Comment:
I don't think we need to know about LandmarkFactory here, only in the .cc file (where the header is included).